src/utils/resolveXYScales.js

```diff
@@ -356,12 +357,15 @@ export default function resolveXYScales(ComposedComponent) {
         if(hasScaleFor(scale, k)) return [k, scale[k]];
 
         // create scale from domain/range
+        // PM/ER December 19, 2016 - Unsure whether or not this is the best approach.
+        //                           Importing d3 in this file also seems wrong. Graphs render correctly.
+        let kScale;
         const rangeMethod = (scaleType[k] === 'ordinal') ? 'rangePoints' : 'range';
-        const kScale = initScale(scaleType[k])
```

Comment:
original comment:
erosenberg 22 hours ago Member
eventually we'll have to redo this but it works for now